src/app.module.ts

```diff
@@ -1,25 +1,69 @@
+import { AssetMetricsController } from './modules/asset-metrics/asset-metrics.controller';
+import { AssetEssentialsRtController } from './modules/asset-essentials-rt/asset-essentials-rt.controller';
+import { AssetEssentialsWrtController } from './modules/asset-essentials-wrt/asset-essentials-wrt.controller';
+
+import { AssetDetailsService } from './modules/asset-details/asset-details.service';
+import { AssetFundamentalsService } from './modules/asset-fundamentals/asset-fundamentals.service';
+import { AssetSegmentsService } from './modules/asset-segments/asset-segments.service';
+import { AssetEssentialsService } from './modules/asset-essentials/asset-essentials.service';
+import { AssetMetricsService } from './modules/asset-metrics/asset-metrics.service';
+import { AssetEssentialsRtService } from './modules/asset-essentials-rt/asset-essentials-rt.service';
+import { AssetEssentialsWrtService } from './modules/asset-essentials-wrt/asset-essentials-wrt.service';
+import { AssetDetailsEntity, AssetEssentialsRealTimeEntity, AssetEssentialsWithoutRealTimeEntity, AssetFundamentalsEntity, AssetMetricsEntity, AssetSegmentsEntity } from 'lib-typeorm';
+import { AssetMetricsModule } from './modules/asset-metrics/asset-metrics.module';
+import { AssetEssentialsRtModule } from './modules/asset-essentials-rt/asset-essentials-rt.module';
+import { AssetEssentialsWrtModule } from './modules/asset-essentials-wrt/asset-essentials-wrt.module';
+
+
 @Module({
   imports: [
-    ConfigModule.forRoot(),
-    TypeOrmModule.forRootAsync({
-      useFactory: getOrmConfig,
+    ConfigModule.forRoot({
+      isGlobal: true,
+    }),
+    TypeOrmModule.forRoot({
+      type: 'postgres',
+      host: process.env.DB_HOST,
+      port: 5432,
+      username: process.env.DB_USERNAME,
+      password: process.env.DB_PASSWORD,
+      database: process.env.DB_DATABASE_NAME,
+      schema: process.env.DB_SCHEMA,
+      entities: [AssetDetailsEntity, AssetEssentialsRealTimeEntity, AssetEssentialsWithoutRealTimeEntity,  AssetSegmentsEntity, AssetFundamentalsEntity, AssetMetricsEntity], 
+      synchronize: true,
+      ssl: true,
+      extra: {
+        ssl: {
+          rejectUnauthorized: false
+        }
+      }
+      // logging: true,
     }),
-    AssetEssentialsModule,
     AssetDetailsModule,
-    AssetSegmentsModule,
     AssetFundamentalsModule,
+    AssetSegmentsModule,
+    //AssetEssentialsModule,
+    AssetMetricsModule,
+    AssetEssentialsRtModule,
+    AssetEssentialsWrtModule,
   ],
-  controllers: [AppController],
+  controllers: [AppController, AssetDetailsController,AssetEssentialsController, AssetFundamentalsController, AssetSegmentsController, AssetMetricsController, AssetEssentialsRtController, AssetEssentialsWrtController],
+  providers: [AssetDetailsService, AssetEssentialsService, AssetFundamentalsService, AssetSegmentsService, AssetMetricsService, AssetEssentialsRtService, AssetEssentialsWrtService],
```

Comment:
I think we should revert the changes on this file. We don't have to import all the modules, controllers, entities and services here. Just the modules are enough in imports section. Also, here just to instantiate a DB is enough, we need to declare the TypeORM entities in their respective modules like `TypeOrmModule.forFeature([xyz.entity.ts])`.
Let me know if there is still an issue.